packages/plugin-conversation/test/integration/spec/verbs.js

```diff
@@ -337,6 +338,34 @@ describe(`plugin-conversation`, function() {
         }));
     });
 
+    describe(`#updateTypingStatus()`, () => {
+      beforeEach(() => spark.conversation.create({participants, comment: `THIS IS A COMMENT`})
+        .then((c) => {conversation = c;}));
+
+      it(`sets the typing indicator for the specified conversation`, () => {
+        const spy = sinon.spy();
+        mccoy.spark.mercury.on(`event:status.start_typing`, spy);
+
+        return spark.conversation.updateTypingStatus(conversation, {typing: true})
+        .then(() => spark.conversation.get(conversation))
```

Comment:
Doing a get on the conversation has no bearing on the test other than to add delay. you probably want to do some similar to the transoding test (search the repo for "transcod", i can never remember if it ends in "ing" or "ed"). basically, wrap the event listener in a promise such that the promise chain is blocked until the event is received.